src/components/dialogmoteunntak/DialogmoteunntakSkjema.tsx

```diff
@@ -86,37 +103,54 @@ const DialogmoteunntakSkjema = () => {
   };
 
   return (
-    <StyledPanel>
+    <Panel style={style.panel}>
       <AlertStripeInfo>{texts.noBrev}</AlertStripeInfo>
       <p>{texts.infoKandidatlist}</p>
-      <Form onSubmit={submit} validate={validate}>
-        {({ handleSubmit }) => (
-          <form onSubmit={handleSubmit}>
-            <SkjemaFieldWrapper>
-              <DialogmoteunntakSkjemaArsakVelger />
-            </SkjemaFieldWrapper>
-            <SkjemaFieldWrapper>
-              <DialogmoteunntakSkjemaBeskrivelse />
-            </SkjemaFieldWrapper>
-            {settDialogmoteunntak.isError && (
-              <SkjemaInnsendingFeil error={settDialogmoteunntak.error} />
-            )}
-            <FlexRow columnGap={FlexGapSize.SM}>
-              <Hovedknapp
-                spinner={settDialogmoteunntak.isLoading}
-                autoDisableVedSpinner
-                htmlType="submit"
-              >
-                {texts.send}
-              </Hovedknapp>
-              <Link to={moteoversiktRoutePath}>
-                <Flatknapp>{texts.avbryt}</Flatknapp>
-              </Link>
-            </FlexRow>
-          </form>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <RadioGroup
```

Comment:
Bør sjekke ut om vi heller bør bruke size="small"-varianten av RadioGroup her. Tror den er ment for interne flater og ligner størrelsemessig mer på radioknappene i det gamle designsystemet.

Reply:
Ja 🙌 Det står også på aksel at `small` er ment for interne flater🙏